build/buildpipeline/pipeline.groovy

```diff
@@ -0,0 +1,11 @@
+import org.dotnet.ci.pipelines.Pipeline
+
+def windowsPipeline = Pipeline.createPipeline(this, 'windows.groovy')
```

Comment:
Ok, learned some things from getting aspnet-docker working. it looks like the second parameter needs to be the file path relative to repo root., so `Pipeline.createPipeline(this, 'build/buildpipeline/windows.groovy')`